opensrp-chw-core/src/test/java/org/smartregister/chw/core/dao/FamilyMemberDaoTest.java

```diff
@@ -43,4 +43,36 @@ public void testGetActiveAlerts() {
         Assert.assertEquals(familyMembers.get(0).getLastName(), "demo_last_name");
     }
 
+    @Test
+    public void testGetMaleFamilyMembers() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`